.env.test

```diff
@@ -38,3 +38,5 @@ export PROCTOR_DOCS_PATH=/path/to/docs/dir
 export PROCTOR_AUTH_PLUGIN_BINARY=
 export PROCTOR_AUTH_PLUGIN_EXPORTED=GateAuth
 export PROCTOR_AUTH_ENABLED=false
+export PROCTOR_NOTIFICATION_PLUGIN_BINARY=
+export PROCTOR_NOTIFICATION_PLUGIN_EXPORTED=SlackNotification
```

Comment:
@walbertus I think we will need to rethink the format for plugins, since Notification plugin here will become observers for the events, so it could be many.

How about if we use Yaml format with Base64 encoding?